### PR TITLE
pr alex

### PR DESCRIPTION
**Pull request by yjwangit**
Hi, would you check it out? Thanks.

### CODE REVIEW

File: eventonica-react/package.json [hunk 1]
```diff
@@ -34,5 +34,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "proxy": "http://localhost:9000"
```
Comment (alexcheva) [inline eventonica-react/package.json]: I'm not sure if we need that? try deleting that and seeing what happens.

**Review by alexcheva — COMMENTED** [review]
delete proxy

File: eventonica-react/src/components/Events.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type':'application/json'
+      },
+      body:JSON.stringify(_events)
+    })
+    let res = await result.json()
+    console.log(res)
+  };
+
```
Comment (alexcheva) [inline eventonica-react/src/components/Events.js:31]: Looks like there is an issue here somewhere.

**Review by alexcheva — COMMENTED** [review]
Ask your mentor to help you to figure it out. Don't forget that we made changes on the "pr" branch when you look at it

File: api/app.js [hunk 3]
```diff
@@ -39,4 +40,29 @@ app.use(function(err, req, res, next) {
...
+  };
+  const result = await db.any("INSERT into events (eventname,eventdate,category,location)VALUES("+_eventsSchema.eventName+","+_eventsSchema.eventDate+","+_eventsSchema.category+","+_eventsSchema.location+")",[true])
+  res.status(200).send({msg:'successful'})
+})
+
+
+app.use(function(req, res, next) {
+  next(createError(404));
+});
+app.listen(9000);
```
Comment (LeiaQuesada) [inline api/app.js:63]: Does this work? Sorry Yingjie..

File: eventonica-react/package.json [hunk 4]
```diff
@@ -34,5 +34,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
```
Comment (LeiaQuesada) [inline eventonica-react/package.json:37]: trailing comma, delete

**Review by LeiaQuesada — COMMENTED** [review]
added some suggestions to review

**Pull request closed without merging**